lib/lhm/sql_retry.rb

```diff
@@ -62,5 +47,36 @@ def default_retry_config
         end
       }.freeze
     end
+
+    def retry_classes_list
+      {
+        "ActiveRecord::LockWaitTimeout" => nil,
+        "ActiveRecord::Deadlocked" => nil,
+	  "ActiveRecord::QueryTimedout" => nil,
```

Comment:
We decided we only want to retry based on the message `Timeout waiting for a response from the last query`